backend/pretix/__init__.py

```diff
@@ -399,11 +411,18 @@ class CreateOrderInput:
     locale: str
     payment_provider: str
     invoice_information: InvoiceInformation
-    tickets: List[CreateOrderTicket]
+    tickets: list[CreateOrderTicket]
 
     def validate(self) -> CreateOrderErrors:
         errors = CreateOrderErrors()
-        self.invoice_information.validate(errors)
+
+        with errors.with_prefix("invoice_information"):
+            self.invoice_information.validate(errors)
+
+        for index, ticket in enumerate(self.tickets):
+            with errors.with_prefix(f"tickets.{index}"):
```

Comment:
maybe this is nicer?

```python
errors.with_prefix("tickets", index)
errors.with_prefix("tickets", index, "another")
```

instead of manually specifying `.` and strings?